app/components/Activity/TransactionModal/TransactionModal.js

```diff
@@ -176,7 +117,9 @@ export default class TransactionModal extends React.PureComponent {
                 <Link
                   className="hint--bottom-left"
                   data-hint={item.block_hash}
-                  onClick={() => blockExplorer.showBlock(network, item.block_hash)}
+                  onClick={() =>
+                    networkInfo && blockExplorer.showBlock(networkInfo, item.block_hash)
```

Comment:
This pattern is reused across this file. May be a good idea to extract it like these for the ease of change and cleanness (untested)
```
const showBlock = (hash) => {
  return networkInfo && blockExplorer.showTransaction(networkInfo, hash)	
} 

```

Reply:
Updated.